.clang-format

```diff
@@ -7,13 +7,12 @@
 
 ---
 Language: Cpp
+Standard: Cpp11
 BasedOnStyle: Google
 AllowShortFunctionsOnASingleLine: None
 AllowShortIfStatementsOnASingleLine: false
 AllowShortLoopsOnASingleLine: false
 BinPackArguments: false
 BinPackParameters: false
 IncludeBlocks: Regroup
-Standard: Cpp11
```

Comment:
?

Reply:
this was moved to be the 2nd line :)

Reply:
yeah but why?^^

Reply:
from above link that was updated:

```
Standard (LanguageStandard)

Possible values:

LS_Cpp03 (in configuration: Cpp03) Use C++03-compatible syntax.
LS_Cpp11 (in configuration: Cpp11) Use features of C++11, C++14 and C++1z (e.g. A<A<int>> instead of A<A<int> >).
LS_Auto (in configuration: Auto) Automatic detection based on the input.
```

Therefore - using Cpp11.